widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -572,13 +572,16 @@ internal class ChatController(
 
     private fun onEngagementStateChanged(state: State) {
         when (state) {
-            State.FinishedCallVisualizer, State.FinishedOmniCore -> {
+            is State.EngagementEnded -> {
                 if (!isQueueingOrOngoingEngagement) {
                     dialogController.dismissDialogs()
                 }
+                if (state.onEnd == Engagement.ActionOnEnd.RETAIN) {
+                    onTransferredToSecureConversation()
+                }
             }
 
-            State.StartedOmniCore -> newEngagementLoaded()
+            is State.EngagementStarted -> newEngagementLoaded()
```

Comment:
To double check, previously `newEngagementLoaded()` was called only for OmniCore. Now it will be called for Call Visualizer as well. Is this an intended change?